modules/cloudwatch-alarm-actions/lambda-subsciptions.tf

```diff
@@ -79,3 +79,30 @@ module "notify_teams" {
     module.topic # TODO: seems there is no need on this dependency, but without this it fails on getting topic by name in underlying subscription module, please check and get right solution of this
   ]
 }
+
+module "notify_cmdb" {
+  source = "./modules/lambda-subscription"
+
+  for_each = { for key, webhook in var.cmdb_webhooks : key => webhook }
+
+  # sns/subscription configs
+  sns_topic_arn    = module.topic.arn
+  sns_topic_name   = module.topic.name
+  sns_subscription = false
+
+  # lambda configs
+  uniq_id   = each.key
+  type      = "cmdb"
```

Comment:
I would suggest to remove this by introducing default values or removing completely.